tests/group_add_mock_test.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/fivetran/go-fivetran"
+	"github.com/fivetran/go-fivetran/tests/mock"
+)
+
+func TestGroupAddUserServiceDo(t *testing.T) {
+	// arrange
+	groupID := "projected_sickle"
+	email := "john.white@mycompany.com"
+	role := "Account Administrator"
+
+	ftClient, mockClient := CreateTestClient()
+	handler := mockClient.When(http.MethodPost, fmt.Sprintf("/v1/groups/%s/users", groupID)).
+		ThenCall(func(req *http.Request) (*http.Response, error) {
+			var requestBody fivetran.GroupAddUserRequest
+			err := json.NewDecoder(req.Body).Decode(&requestBody)
```

Comment:
Decode it here to map[string]interface{} to avoid breaking incapsulation.
You've already done the same in group create test:
```
body := requestBodyToJson(t, req)
assertGroupCreateRequest(t, body)
```